### ISSUE TITLE
Can I analyse genre of my own audio file using this?

### ISSUE THREAD

**Issue by gingerbeardman** [Mithgroth/music-genre-recognition-gtzan#1]
Can I analyse genre of my own audio file using this?

Thanks!

**Comment by Mithgroth** [comment]
Hey, sorry for late response.
This is part of the series of me transitioning into machine learning from .NET backend - so this is mostly an experiemental repository, and it's still ongoing.

Having said that, thus far results are interesting as 3 seconds samples average better F1-scores, compared to 30 seconds training set. Mind you this is based on GTZAN dataset with 20% validation set split.

I'm very curious how it'll perform on your samples, please let me know!

The GTZAN dataset also has images of audio files, I'm really curious to see how that performs on the validation set, and maybe even on other audio files!

Another note would be: Since this is experimental and ongoing - trainer is nowhere near optimized. But in time, I'm hoping to add things as my own ML journey continues.

Cheers for stopping by :)
